ruby-challenges.rb

```diff
@@ -3,32 +3,87 @@
 
 # --------------------1) Create a method that takes in a number and determines if the number is even or odd. Use the test variables provided.
 
+# A method that takes in a number
+def even_or_odd num1
+    # if number is even (num1 % 2 = 0), return it is even
+    if num1 % 2 == 0
```

Comment:
Awesome work! I would recommend naming your parameter something different than the variable.  Also, for fun take a look at the Ruby built in methods .even? and .odd?